MobiFlight/Joysticks/JoystickManager.cs

```diff
@@ -178,11 +177,6 @@ private bool HasAxisOrButtons(Joystick js)
                 js.Capabilities.ButtonCount > 0;
         }
 
-        private void Js_OnAxisChanged(object sender, InputEventArgs e)
```

Comment:
This should not get lost, the onButtonPress mapping should probably be done outside of the Joystick Manager in the Execution Manager.

Reply:
Added extra JoystickDeviceType.